CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
 ### 🎁 New Features
 
 * `MenuItem` now supports a `className` prop.
+* `TrackService` now posts `url` and `appVersion` fields to the server when tracking an event.
+* `ActivityTrackingModel` now posts `filterChooserModel` filter specs to server for server-side
```

Comment:
I would write this to be a little less technical, and a little more user targeted e.g. "Admin `Activity` and `Errors` modules now use server-side filtering for better support of large datasets."